src/alpaka/AlpakaDataFormats/BeamSpotAlpaka.h

```diff
@@ -0,0 +1,29 @@
+#ifndef AlpakaDataFormats_BeamSpot_interface_BeamSpotAlpaka_h
+#define AlpakaDataFormats_BeamSpot_interface_BeamSpotAlpaka_h
+
+#include "AlpakaCore/alpakaCommon.h"
+#include "DataFormats/BeamSpotPOD.h"
+
+#include <cstring>
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  class BeamSpotAlpaka {
+  public:
+    BeamSpotAlpaka() = default;
+
+    BeamSpotAlpaka(BeamSpotPOD const* data, Queue& queue) : data_d{cms::alpakatools::allocDeviceBuf<BeamSpotPOD>(1u)} {
+      auto data_h{cms::alpakatools::createHostView<const BeamSpotPOD>(data, 1u)};
+
+      alpaka::memcpy(queue, data_d, data_h, 1u);
```

Comment:
I suppose the `data_h` does not need to be alive when the asynchronous operation launched by `alpaka::memcpy` completes?

Reply:
Yes I am not sure about this case. 
Usually `alpaka::wait(queue);` is needed at end of scope, because otherwise the Queue object / host & device buffers objects and their data would be destructed.

Here it is not obvious, because the `queue` is not destructed at end of scope, `data_d` owns the device data for entire`BeamSpotAlpaka` lifetime, and data is not owned by `data_h `anyway (it is just a view). 

So it likely works anyway, but I presume in client code, if the `BeamSpotAlpaka` object is destructed before a `alpaka::wait(queue);` we run into trouble.

Added the `alpaka::wait(queue);`  for safety. If anything, studying potential removals of  `alpaka::wait(queue);` will be completely straightforwards, will dealing with the segfaults turns out way more complicated.